pandas/tests/indexing/test_datetime.py

```diff
@@ -51,15 +51,15 @@ def test_indexing_with_datetime_tz(self):
         # indexing
         result = df.iloc[1]
         expected = Series(
-            [Timestamp("2013-01-02 00:00:00-0500", tz="US/Eastern"), np.nan, np.nan],
```

Comment:
does this not work now (the existing code)?

Reply:
I think the test is incorrect, and tm.assert_series_equal is getting it wrong.
```
now = pd.Timestamp.now("US/Eastern")
left = pd.Series([now, pd.NaT, pd.NaT], dtype=object)
right = pd.Series([now, np.nan, np.nan], dtype=object)

tm.assert_series_equal(left, right)
```

Reply:
Will open an issue for assert_series_equal